### ISSUE TITLE
čudny kalendar

### ISSUE THREAD

**Issue by milandzuris** [bruxy70/Garbage-Collection#353]
### Before you submit a new bug report, please check that

- [X] I have read [frequently asked questions and limitations](https://github.com/bruxy70/Garbage-Collection/wiki/Review-before-submitting-a-new-issue-or-feature-request)
- [X] I'm running the newest version of Garbage Collection, or the next pre-release.
- [X] I have enabled debug logging for my installation.

### Describe the bug

Mam otazku ako pridat takyto zberny kalendar ktory sa presne neopakuje, až na tie čierne farby červena modra a žlta sa neda nastavit lebo nechodia ani každy pondelok a ani prvy pondelok v mesiace, jedine čo mi ostava pridat všetko po jednom až na čiernu farbu, ale potom mam plno senzory viem to dajak spravit?, dakujem pekne.
![image](https://user-images.githubusercontent.com/42905545/153455872-297b9b7a-6b70-4ae2-849b-16d0a62ad331.png)

### Configuration

```JSON
.
```

### Debug logs

```shell
.
```

**Comment by bruxy70** [comment]
Ano, jde. Pres ten include blueprint. Tam se daji dat datumy, jeden na radku
```
2022-01-10
2022-02-07
2022-03-21
```
atd...

Psal jsem to vyse.

**Comment by milandzuris** [comment]
![image](https://user-images.githubusercontent.com/42905545/153471927-6700898d-b387-45f8-a700-5ed5a4128721.png)
robim niečo zle že to nechce zmenit? 
![image](https://user-images.githubusercontent.com/42905545/153472019-a2148f22-44a9-4399-977d-14f930e424c5.png)
dal som aj run action a nič, dakujem.

**Comment by bruxy70** [comment]
https://github.com/bruxy70/Garbage-Collection/wiki/Review-before-submitting-a-new-issue-or-feature-request
